README.md

```diff
@@ -190,12 +189,13 @@ kind: Application
 metadata:
   name: "wordpress-01"
   labels:
-    app.kubernetes.io/name: "wordpress-01"
+    app.kubernetes.io/name: "wordpress"
+    app.kubernetes.io/instance: "wordpress-01"
     app.kubernetes.io/version: "3"
 spec:
   selector:
     matchLabels:
-     app.kubernetes.io/name: "wordpress-01"
+     app.kubernetes.io/instance: "wordpress-01"
```

Comment:
The uniqueness could be name+instance. there's not recommendation around that point 
e.g
```
app.kubernetes.io/name: "wordpress"
app.kubernetes.io/instance: "01"
```

I would match labels both.